### ISSUE TITLE
Element-wise mean

### ISSUE THREAD

**Issue by rafaelanchieta** [bentrevett/pytorch-pos-tagging#7]
Hi, Ben Trevett.

How can I calculate the element-wise mean on the outputs of a BiLSTM and to feed that result in another BiLSTM?

`out, (hidden, cell) = self.bilstm(in)`
`element = torch.mean(out, keepdim=True, dim=1)`
`out2, (_, _) = self.bilstm2(element)`

Is that correct?

**Comment by bentrevett** [comment]
If your LSTMs have `batch_first=False` (the default) then your `torch.mean` is averaging over the batch dimension, which is probably not what you want.

Why do you want to do this? I can maybe see a case of averaging `out` across the `seq_len` dimension to use as the initial hidden state of `bilstm2`, but I can't see why you would want to use it as the input to a second LSTM.

**Comment by rafaelanchieta** [comment]
Thank you for your response.

I'm trying to implement the architecture of the figure.
![architecture](https://user-images.githubusercontent.com/2988889/107350213-e93b6a80-6aa7-11eb-99cb-fe8f8699c334.PNG)

Could you help me how implement that?
